iotdb/iotdb/conf/iotdb-engine.properties

```diff
@@ -57,8 +57,8 @@ force_wal_period_in_ms=10
 ####################
 ### Timestamp Precision Configuration
 ####################
-
-# Use this value to set timestamp precision as millisecond, microsecond or nanosecond.
+# Use this value to set timestamp precision as "ms", "us" or "ns".
+# Once the precision is been set, it can not be changed.
```

Comment:
Will you check whether the precision has been changed since the last system start? And if it does be changed, will the user get an error or something?